aliases: cleanups and keep-sorted

qweered opened the pull request:
- [x] Fits CONTRIBUTING.md, pkgs/README.md, maintainers/README.md and other READMEs.

wolfgangwalther commented on pkgs/top-level/aliases.nix:
```diff
@@ -368,51 +368,49 @@ mapAliases {
   forceSystem = system: _: (import self.path { localSystem = { inherit system; }; });
 
-  ### _ ###
+  # When the nixops_unstable alias is removed, nixops_unstable_minimal can be renamed to nixops_unstable.
```
Let's remove this outdated comment in a commit before (again, to make the sorting commit entirely reproducible)

wolfgangwalther reviewed (commented):
Left a few comments, the pattern should be clear: Let's move all manual modifications into commits before the keep-sorted one, so that we can easily recreate the latter.

qweered committed:
aliases: remove outdated and header comments

These header comments won't be useful anymore with keep-sorted sorting,
while the comment about nixops is just outdated.

qweered committed:
aliases: remove unsorted section and move preserve to the top

qweered committed:
aliases: remove inherit

qweered committed:
aliases: keep-sorted

qweered commented:
Done

qweered committed:
.git-blame-ignore-revs: add keep-sorted aliases commit

qweered committed:
aliases: convert '' to " in throws

emilazy reviewed (commented):
Overall neutral on this. Some of the organization this disrupts is intentional / something I’ve found useful, but OTOH there are a lot of arbitrary weirdnesses in the file too. The `aliases.nix` file is somewhat unmaintainable with the current system in general, so I guess I’m really fine with any ordering :)

The general clean‐ups are definitely good, and may help with a final round of dropping with https://github.com/NixOS/nixpkgs/pull/427017.

cc @wolfgangwalther – do we want to land this? I think I’ve talked myself into it being a good idea while writing this.

qweered commented:
Note: throw '' '' (aka multiline) need to be replaced with default throw " " for keep sorted to work

wolfgangwalther reviewed (approved):
Good stuff, thank you!

I went through the commits, adjusted the commit messages a bit and move some more "preserve" aliases to the top in the second commit. I also dropped the commit that changed `''` to `"` for multi-line messages, because I believe this changes the messages contents.

With the split, I was able to recreate the last commit easily, nice.

(I also adjusted the commit hash in .git-blame-ignore-revs after my rebase)

qweered commented:
Working on fix
